arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -1718,6 +1718,17 @@
               "format": "date-time",
               "description": "UTC date and time when the process started"
             },
+            "role": {
+              "type": "string",
```

Comment:
Are you sue that this is the definitive enum list? If this enum list is expected to change in the future, it is recommended we use x-ms-enum and set set modelAsString to true. Please refer this[https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/creating-swagger.md#enums-with-x-ms-enum]